src/widgets/StackedTables.js

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable react/forbid-prop-types */
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2019
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { StyleSheet, View } from 'react-native';
+
+import MiniTable from './SimpleTable';
```

Comment:
Why not just import it as SimpleTable? I haven't done ES6 for so long I can't remember if you have to do more than this to alias a default import lol.

Reply:
Mistake, leftover from renaming and messy dev branch not trying to do anything tricky. Done 3486dd3a76dcdd68eefe62a2aabcd92d980bba34